WordPress/src/main/res/layout/login_2fa_screen.xml

```diff
@@ -64,10 +64,12 @@
                     style="@style/Base.TextAppearance.AppCompat.Caption"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
+                    android:minHeight="48dp"
```

Comment:
Great point about the `dimen`! Now that you mention it, funny we didn't already have that! Will introduce that! Done with 633d3d6.

About the visual feedback on tap, let's defer that to the visual design phase. I'm actually thinking that we could probably use the borderless Material button there directly. Thanks for the note!